modules/osdk-init-resource.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-generating-csvs.adoc
+
+[id="osdk-init-resource_{context}"]
+= Initializing required custom resources
```

Comment:
This module title makes it sound like a procedure, but there are no explicit steps. Either rework the content into a procedure or retitle the module. Maybe "Operators that require custom resources"

Reply:
Was matching the "Hiding internal objects" that preceded it, however I've now rewritten both modules to have a one-step procedure.